config.ts

```diff
@@ -5,7 +5,7 @@ export enum Network {
   MUMBAI = 'MUMBAI',
 }
 
-export const NETWORK_CONFIG = {
+export const NETWORK_CONFIGS = {
```

Comment:
suggestion: use type at top level
```suggestion
export const NETWORK_CONFIGS: {
  [key in Network]: NetworkConfig
} = {
```